### DISCUSSION TITLE
Mark the reviews/comments of external people as "External"

### DISCUSSION THREAD

**Discussion by aminya** [community/community#37745] [Pull Requests]
### Feature Request

When some external person leaves a review/comment in a pull request or issue, their comment should be marked as "External". 

### Reason

This solves an issue with external people requesting things that are not needed by the actual owner/maintainers of the project. 
People think the request of external people is a requirement by the owner/organization, and it sometimes results in serious disputes that affect the governance of the project.

Another issue is that GitHub currently marks someone who made one commit to a repository as a "contributor". However, this is confusing and often misinterpreted. 

If GitHub marks the comments of any non-member/maintainer explicitly as "External", such problems will be avoided.

**Comment by ossanna16** [comment]
Hi there @aminya and welcome to our community! Thank you for the feature request!

**Reply by aminya** [reply]
Thank you. You're welcome
